CHANGELOG.md

```diff
@@ -3,6 +3,205 @@
+  would allow it to be used even if `--no-managed-python` was provided. Now, uv ignores Python
+  interpreters that do not conform to the Python preference _unless_ they are in an active virtual
+  environment or are explicitly requested, e.g., with `--python /path/to/python3.13`.
+
+  Similarly, uv would previously not invalidate existing project environments if they did not match
+  the Python preference. Now, uv will invalidate and recreate project environments when the Python
+  preference changes.
+
+  You can opt-out of this behavior by providing the explicit path to the Python interpreter.
+
+- **Install dependencies without build systems when they are `path` sources ([#14413](https://github.com/astral-sh/uv/pull/14413))**
+
+  When working a project, uv uses the [presence of a build
+  system](https://docs.astral.sh/uv/concepts/projects/config/#build-systems) to determine if it
+  should be built and installed into the environment. However, when a project is a dependency of
+  another project, it can be surprising for the dependency to be missing from the environment. Now,
+  dependencies with [`path` sources](https://docs.astral.sh/uv/concepts/projects/dependencies/#path)
+  are built and installed, regardless of the presence of a build system. If a build system is not
+  present, the `setuptools.build_meta:__legacy__ ` backend will be used (per the standards).
+
+  You can opt-out of this behavior by setting `package = false` in the source declaration.
+
+  See the documentation on [virtual
+  dependencies](https://docs.astral.sh/uv/concepts/projects/dependencies/#virtual-dependencies) for
+  details.
+
+- **Install dependencies without build systems when they are workspace members ([#14663](https://github.com/astral-sh/uv/pull/14663))**
+
+  As with the change for dependencies with `path` sources, uv previously would not build and install
+  workspace members that did not declare a build system. Now, uv will build and install workspace
+  members that are a dependency of _another_ workspace member regardless of the presence of a build
+  system. The behavior is unchanged for workspace members that are not included in the
+  `project.dependencies`, `project.optional-dependencies`, or `dependency-groups` tables of another
+  workspace member.
+
+  You can opt-out of this behavior by setting `tool.uv.package = false` in the workspace member's
+  `pyproject.toml`.
+
+  See the documentation on [virtual
+  dependencies](https://docs.astral.sh/uv/concepts/projects/dependencies/#virtual-dependencies) for
+  details.
+
+- **Bump `--python-platform linux` to `manylinux_2_28` ([#14300](https://github.com/astral-sh/uv/pull/14300))**
+
+  Previously, the default target for `--python-platform linux` was `manylinux_2_17`. Now, the
+  default is `manylinux_2_28`.
+
+  Only [0.2% of packages require `manylinux_2_17`](https://mayeut.github.io/manylinux-timeline/)
```

Comment:
"packages" is incorrect here, the chart is about clients making request to PyPI. How about either "0.3% of PyPI users require `manylinux_2_17`" or "0.3% of requests to PyPI come from a system that is only compatible with `manylinux_2_17` and not newer versions"?

(also the number bumped to 0.3% since Charlie opened the PR)

Reply:
Thanks!

Reply:
I wonder if I should just drop these statistics.

There are both consumer and package statistics at https://mayeut.github.io/manylinux-timeline/

Reply:
I'd tend towards keeping this section short, reflecting that we expect it to have lower impact than other changes in this release.